docs/faq.asciidoc

```diff
@@ -75,48 +102,51 @@ A good rule of thumb is that `queue.mem.events` should equal `output.elasticsear
 
 [float]
 [[increase-cluster-ingest]]
-==== Tune Elasticsearch for higher ingestion
+=== How to tune Elasticsearch for higher ingestion?
 
 Increasing the Elasticsearch ingestion rate is a large topic.
 See {ref}/tune-for-indexing-speed.html[Tune for indexing speed],
 particularly the sections about how to increase the refresh interval,
 disable swapping, use faster hardware, and set the indexing buffer size.
 
-[float]
-[[reduce-data]]
-==== Reduce the amount of data collected
-
-The most obvious way to reduce the number of documents to be indexed
-is to reduce the _transaction sample rate_.
-The transaction sample rate is controlled in the configuration of agents (for example for {apm-py-ref}/configuration.html#config-transaction-sample-rate[Python] and {apm-node-ref}/agent-api.html#transaction-sample-rate[Node.js]).
-
-Reducing the transaction sample rate does not affect the collection of metrics such as Requests Per Second.
-
 [float]
 [[increase-queue-size]]
-==== Increase internal queue size
+=== How to increase internal queue size?
 
 A larger internal queue allows Elasticsearch to be unavailable for longer periods,
 and it alleviates problems that might result from sudden spikes of data.
 You can increase the queue size by overriding `queue.mem.events`.
 Be aware that increasing `queue.mem.events` can significantly affect APM Server memory usage.
 
+[float]
+[[add-apm-server-nodes]]
+=== How to add APM Server nodes?
+
+When the APM Server cannot process incoming requests quickly enough,
+you will see "503: Request timed out waiting to be processed" messages.
+
+The best way to avoid this problem is to add more processing power to your APM Server cluster.
+You can either migrate your APM Server processes to more powerful machines or add more machines.
+
 [float]
 [[delete-old-data]]
-==== Delete old data
+=== How to delete old data?
 
-The internal queue might be full because Elasticsearch ran out of disk space and started rejecting insertions.
-If this happens,
-you will need to remove old indices to make room for new data.
-To do this you can use a tool like Curator and set up a cron job to run it periodically.
+It might make sense to delete old APM indices on a regular basis to make room for new data. 
+To do this you can use a tool like {curator-ref-current}[Curator] and set up a cron job to run it periodically.
 
 By default APM indices have the pattern `apm-%{[beat.version]}-{type}-%{+yyyy.MM.dd}`.
 With the curator command line interface you can, for instance, see all your existing indices:
 
-["source","sh",subs="attributes"]
+["source","sh"]
 ------------------------------------------------------------
 curator_cli --host localhost show_indices --filter_list '[\{"filtertype":"pattern","kind":"prefix","value":"apm-"\}]'
+------------------------------------------------------------
+// CONSOLE
```

Comment:
Unfortunately // CONSOLE only works on REST calls, so you should remove it here. It would be *really cool* if we had similar functionality to open commands in a terminal window, but we don't have that right now.